src/components/Form/Form.stories.mdx

```diff
@@ -0,0 +1,130 @@
+import {
+  Meta,
+  Story,
+  Canvas,
+  ArgsTable,
+  Preview,
+  Description,
+} from "@storybook/addon-docs/blocks";
+import { withDesign } from "storybook-addon-designs";
+import { getCategory } from "../../utils/componentCategories";
+
+import Form, { FormRow, FormField } from "./Form";
+import TextInput from "../TextInput/TextInput";
+import Radio from "../Radio/Radio";
+import Checkbox from "../Checkbox/Checkbox";
+import Label from "../Label/Label";
+import Button from "../Button/Button";
+import Select from "../Select/Select";
+import HelperErrorText from "../HelperErrorText/HelperErrorText";
+import { action } from "@storybook/addon-actions";
+
+<Meta
+  title={getCategory("Form")}
+  component={Form}
+  decorators={[withDesign]}
+  parameters={{
+    design: {
+      type: "figma",
+      url:
+        "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Master?node-id=11895%3A547",
+    },
+  }}
+/>
+
+# Form
+
+<Description of={Form} />
+
+The `Form` component renders a standard `<form>` element and should be used to handle spacing for child input fields. Use the `FormRow` and `FormField` child-components to arrange your input fields accordingly.
+
+Multiple `FormField` components can be placed in a parent `FormRow` component. However, a `FormField` component does not have to placed within a `FormRow` component. `FormField` components can stand alone within `Form`.
```

Comment:
When should `FormField` components not be in `FormRow` components? If there's only one row?

Reply:
I'm trying to figure out if there's a way to make these divs not so deeply nested.  
For the `<textInput>`, which is a wrapper around the label and input, there's really no need for both a `<FormField>` and a `<TextInput>`  It makes more sense to just have a <FormRow> that handles box spacing and stuff.  

But I do see why `<FormField>` is necssary in the checkbox and radio groups - I wonder if it makes more sense to make `FormRow` required, and `FormField` optional.

Reply:
I understand what you are saying about div wrappers around the form input components.  We are not there yet, but we can get there.  For now, however, we will continue to use `FormField` and `FormRow`.  I rewrote the description trying to clarify how the child components should be used.